sks_nodepool: fix "instance_type" attribute

falzm opened the pull request:
This change fixes the `exoscale_sks_nodepool.instance_type` resource
attribute to support non-standard Compute instance types.

greut commented:
the SSH Key pair is not properly deleted from the acceptance tests... investigate!

Commit:
tests: refactor validation helper functions

Commit:
sks_nodepool: fix "instance_type" attribute

This change fixes the `exoscale_sks_nodepool.instance_type` resource
attribute to support non-standard Compute instance types.
